lms/envs/aws.py

```diff
@@ -108,6 +108,10 @@
 # don't know what that call is or if it's active at this point in the code.
 if os.environ.get('QUEUE') == 'high_mem':
     CELERYD_MAX_TASKS_PER_CHILD = 1
+# ACE needs to be treated similarly, see EDUCATOR-2456 for more details
+if os.environ.get('QUEUE') == 'ace':
```

Comment:
`high_mem` and `ace` are both defined as `queue` entries here: https://github.com/edx/edx-internal/blob/650988bc133b9a202c668cfb49df1ed76714395e/ansible/vars/edx.yml#L643-L684